src/components/Spinner/index.js

```diff
@@ -1,18 +1,46 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import AssistiveText from './../AssistiveText';
-import StyledSpinner from './styled/spinner';
-
+import StyledCircleSpinner from './styled/circleSpinner';
+import StyledArcSpinner from './styled/arcSpinner';
+import StyledSpinnerContainer from './styled/spinnerContainer';
+import StyledChildContainer from './styled/childContainer';
+import getSizeValue from './helpers/getSizeValue';
 /**
  * Spinners should be shown when retrieving data or performing slow,
  * help to reassure the user that the system is actively retrieving data.
  */
 export default function Spinner(props) {
-    const { className, style, assistiveText, isVisible, size, variant } = props;
+    const { className, style, assistiveText, isVisible, size, variant, type, children } = props;
+    const currentSize = getSizeValue(size);
 
     if (isVisible) {
+        if (type === 'arc') {
+            return (
+                <StyledSpinnerContainer>
```

Comment:
we need to pass className and style prop to the container, all component containers receive these props in order to pass styles to positioning, margin and other styles they want to pass